docs/src/snap/howto/cis-hardening/etcd.md

```diff
@@ -0,0 +1,198 @@
+<!-- markdownlint-disable -->
+## Etcd Node Configuration
+### Etcd Node Configuration
```

Comment:
why is this title a repeat of the above?

Reply:
The subtitle is sometimes the same as the main title. I added an if statement that doesn't render the subtitle if it's the case.